angular/projects/spark-angular/src/lib/components/sprk-tabs/sprk-tabs.stories.ts

```diff
@@ -45,15 +39,69 @@ only one content panel will be visible at a time.
 };
 
 const modules = {
+  imports: [SprkTabsModule, SprkTabsPanelModule, SprkTabsButtonModule],
+};
+
+export const defaultStory = () => ({
+  moduleMetadata: modules,
+  template: `
+    <sprk-tabs idString="tabs-1">
+      <button
+        sprkTabsButton
+        analyticsString="Tab: 1"
+        data-id="tab-1"
```

Comment:
The data-id here is set as an attribute when the directive has an Input for this. Please update this to use the input.